src/components/transport_manager/src/transport_manager_impl.cc

```diff
@@ -711,10 +711,9 @@ void TransportManagerImpl::CreateWebEngineDevice() {
 
   ws_device->set_keep_on_disconnect(true);
 
-  RaiseEvent(&TransportManagerListener::OnDeviceAdded, web_engine_device_info_);
-  device_list_.push_back(
-      std::make_pair(web_socket_ta, web_engine_device_info_));
   web_socket_ta->AddDevice(ws_device);
+  OnDeviceListUpdated(web_socket_ta);
+  web_socket_ta->ConnectDevice(unique_device_id);
```

Comment:
@AKalinich-Luxoft consider `ConnectAllDevices()` function instead

Reply:
Done in https://github.com/smartdevicelink/sdl_core/pull/3339/commits/54d3c07aeb18dcc52a6bee741efcee9b307a81c6